omise/omise.php

```diff
@@ -290,6 +331,32 @@ public function install()
         return true;
     }
 
+    /**
+     * A PrestaShop version specific version of 'display'
+     *
+     * @return see parent 'display' method
+     */
+    protected function versionSpecificDisplay($file, $template)
+    {
+        $versionedTemplate = PRESTASHOP_VERSION_VIEW_PATH . $template;
+        return $this->display($file, $versionedTemplate);
```

Comment:
@jonrandy maybe `$versionedTemplate` variable is not really necessary here? How about just simply return
```php
return $this->display($file, PRESTASHOP_VERSION_VIEW_PATH . $template);
```

Reply:
OK